### DISCUSSION TITLE
Activate issues here?

### DISCUSSION THREAD

**Discussion by 12people** [sendbird/sendbird-uikit-flutter#12] [Ideas]
It'd be nice to be able to report issues directly for this specific package here.

**Comment by 280027552** [comment]
When using a Super Group Channel, the reply feature is not available in the Flutter SDK, but it works fine in both the iOS and Android SDKs. Please fix this issue as soon as possible.
version：1.0.3

**Comment by 280027552** [comment]
he Flutter SDK lacks many of the customizable features available in the native iOS and Android SDKs. Please prioritize feature parity and improve the functionality as soon as possible.
